### ISSUE TITLE
autopull of sex-biased information

### ISSUE THREAD

**Issue by arw36** [CEIDatUGA/COVID-19-DATA#56]
- [ ] the first table in this website https://globalhealth5050.org/covid19/. 
- [ ] third table from DC's .xlsx file (https://dcgov.app.box.com/v/DCHealthStatisticsData)
- [ ] Utah sex/age group plot that we had looked at earlier https://coronavirus-dashboard.utah.gov/#demographics

**Comment by casanchez** [comment]
First table has moved here https://globalhealth5050.org/covid19/sex-disaggregated-data-tracker/

**Comment by mvevans89** [comment]
woah sorry I completely did not see this issue until that new message. Would you all like me to set an autopull of those datasets?

**Comment by casanchez** [comment]
yes that would be super helpful! I would also be interested in learning how that is done technically

**Comment by mvevans89** [comment]
Okay what I have been doing is adapting peoples' code that scrapes the data and setting it up to run automatically. All of these data sources seem to be embedded in javascript somehow so it's unclear if we can even easily get to the data behind the tables/graphs in an automated way, but I do think it will be a bit of a challenge if it is possible. 

I don't currently have time to do this, but am happy to automate things once you have things working.

**Comment by arw36** [comment]
I added this more of a data tracking issue, and if others have suggestions for automation of manual data pulling. Perhaps more appropriate to put into the covid19_master_datasheet? Haven't had time to dig in to extract data.
